api-reference/10 UI Components/dxTabs/1 Configuration/stylingMode.md

```diff
@@ -5,7 +5,15 @@ default: 'primary'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies an active tab styling mode.
```

Comment:
I generally try to avoid noun phrases longer than three words. And I would use "the", because that's what we usually do for property values. We specify "the control's width", or "the number of items". When we talk about what attribute a property specifies, we generally use "the". (Or sometimes you can get away without an article, like in examples below.)

```suggestion
Specifies the styling mode for the active tab.
```